jgrapht-core/src/main/java/org/jgrapht/util/ConcurrentUtil.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * (C) Copyright 2020-2020, by Semen Chudakov and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * See the CONTRIBUTORS.md file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the
+ * GNU Lesser General Public License v2.1 or later
+ * which is available at
+ * http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR LGPL-2.1-or-later
+ */
+package org.jgrapht.util;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Utility class to manage creation and shutting down instance
+ * if the {@link ThreadPoolExecutor}.
+ */
+public class ConcurrentUtil {
```

Comment:
What do you think about naming this class `ConcurrencyUtil` instead of `ConcurrentUtil`?
The point of the class is providing utilities in the field of concurrency not being itself concurrent (like for example `ConcurrentHashMap` or similar.

Reply:
Good point, I will make the suggested change.

Reply:
Done.